server/notification-providers/wecom.js

```diff
@@ -30,26 +31,50 @@ class WeCom extends NotificationProvider {
     /**
      * Generate the message to send
      * @param {object} heartbeatJSON Heartbeat details (For Up/Down only)
+     * @param {object} monitorJSON Monitor details
      * @param {string} msg General message
      * @returns {object} Message
      */
-    composeMessage(heartbeatJSON, msg) {
-        let title;
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === UP) {
-            title = "UptimeKuma Monitor Up";
-        }
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === DOWN) {
-            title = "UptimeKuma Monitor Down";
+    composeMessage(heartbeatJSON, monitorJSON, msg) {
+        let params = {};
+        if (heartbeatJSON != null) {
+            params = {
```

Comment:
assigning this to a vairable not nessesary
```suggestion
        if (heartbeatJSON != null) {
            return {
```

Reply:
yeah, I don't expect this level of examination of codes.